Software-Engineerin-I2-Team-9/mapquester
# Setup GitHub Actions for Frontend CI/CD

# Frontend CI/CD Setup

## Changes Made:
- Added GitHub Actions workflow configuration
- Setup frontend unit testing with Jest
- Configured ESLint for code quality checks
- Added test files for components:
  - MapComponent tests
  - POIForm tests
  - Home page tests
- Setup test coverage reporting

## How to Test:
- The CI checks should run automatically on this PR
- You can verify frontend tests locally:
  ```bash
  cd mapquester_frontend
  npm run test:ci
